static_src/main.js

```diff
@@ -0,0 +1,82 @@
+
+import '../node_modules/bootstrap/dist/css/bootstrap.css';
+
+import {Router} from 'director';
+import React from 'react';
+
+import App from './app.jsx';
+import cfApi from './util/cf_api.js';
+import Login from './components/login.jsx';
+import orgActions from './actions/org_actions.js';
+import Space from './components/space.jsx';
+import SpaceList from './components/space_list.jsx';
+
+const mainEl = document.querySelector('.js-app');
+
+
+function login() {
+  React.render(<App><Login/></App>, mainEl);
+}
+
+function dashboard() {
+  React.render(<App>
+    <h3>Welcome to CF-Deck</h3>
+    <h5>Pick an organization to get started</h5>
```

Comment:
A new thing to think about maybe for the future. Let's say someone gets an account created via our self-service thing (This actually happened) and they have 0 orgs since they did that. Should have an ```
if orgs.count <= 0 { "You have zero orgs. Please talk to your org manager to get started")}
else {"Pick an organization to get started"}